terraform/azure/main.tf

```diff
@@ -41,3 +55,105 @@ resource "azurerm_storage_account" "orsoarpadev" {
     environment = "dev"
   }
 }
+
+resource "azurerm_app_service_plan" "orsoarpadev" {
+  name                = module.app_service_plan_name.result
+  location            = azurerm_resource_group.orsoarpadev.location
+  resource_group_name = azurerm_resource_group.orsoarpadev.name
+
+  sku {
+    tier = "Free"
+    size = "F1"
+  }
+
+  tags = {
+    environment = "dev"
+  }
+}
+
+resource "azurerm_app_service" "orsoarpadev" {
+  name                = module.app_service_name.result
+  location            = azurerm_resource_group.orsoarpadev.location
+  resource_group_name = azurerm_resource_group.orsoarpadev.name
+  app_service_plan_id = azurerm_app_service_plan.orsoarpadev.id
+  https_only = true
+
+  site_config {
+    dotnet_framework_version = "v5.0"
+    scm_type                 = "GitHub"
+    http2_enabled = true
+    health_check_path = "/health"
+  }
+
+  app_settings = {
+    "Logging:IncludeScopes" = "false"
+    "Logging:LogLevel:Default" = "Trace"
+    "Logging:LogLevel:Microsoft" = "Warning"
+    "Logging:LogLevel:Microsoft.Hosting.Lifetime" = "Information"
+    "Logging:LogLevel:Microsoft.EntityFrameworkCore.Database.Command" = "Information"
+    "EmailConfiguration:From" = "dev@arpa.orso.co"
+    "EmailConfiguration:SmtpServer" = "localhost"
+    "EmailConfiguration:Port" = "25"
+    "EmailConfiguration:Username" = ""
+    "EmailConfiguration:Password" = ""
+    "EmailConfiguration:DefaultSubject" = "Message from ARPA"
+    "JwtConfiguration:TokenKey" = ""
+    "JwtConfiguration:Issuer" = "https://localhost:5001"
+    "JwtConfiguration:Audience" = "https://localhost:5001"
+    "JwtConfiguration:AccessTokenExpiryInMinutes" = "10"
+    "JwtConfiguration:RefreshTokenExpiryInDays" = "3"
+    "IdentityConfiguration:LockoutExpiryInMinutes" = "10"
+    "IdentityConfiguration:MaxFailedLoginAttempts" = "3"
+    "IdentityConfiguration:EmailConfirmationTokenExpiryInDays" = "3"
+    "IdentityConfiguration:DataProtectionTokenExpiryInHours" = "2"
+    "CorsConfiguration:AllowedOrigins:0" = "http://localhost:4200" // ToDo: Add storage url
+    "ClubConfiguration:Name" = "ORSO – Orchestra & Choral Society Freiburg | Berlin e. V."
+    "ClubConfiguration:Address" = "Schwarzwaldstr. 9-11, 79117 Freiburg"
+    "ClubConfiguration:Email" = "mail@orso.co"
+    "ClubConfiguration:Phone" = "+4907617073203"
+    "LocalizationConfiguration:DefaultCulture" = "en-GB"
+    "LocalizationConfiguration:SupportedUiCultures:0" = "en"
+    "LocalizationConfiguration:SupportedUiCultures:1" = "en-GB"
+    "LocalizationConfiguration:SupportedUiCultures:2" = "de"
+    "LocalizationConfiguration:SupportedUiCultures:3" = "de-DE"
+    "LocalizationConfiguration:FallbackToParentCulture" = "true"
+    "APPINSIGHTS_INSTRUMENTATIONKEY" = "" // ToDo: Take from app insights resource
```

Comment:
May be we can integrate app_insights in to app_service as part of a separate PR.